keda/Chart.yaml

```diff
@@ -8,11 +8,11 @@ kubeVersion: ">=v1.23.0-0"
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.12.0
+version: 2.13.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 2.12.0
+appVersion: 2.13.0
```

Comment:
Please revert this and remove the new package; this is only done during releases